src/components/notifications/NotificationsMenu.js

```diff
@@ -201,52 +204,61 @@ function NotificationsMenu(props) {
                 >
                     {t("notifications")}
                 </Typography>
-                {isMobile && [
-                    <NotificationsListingLink
-                        key={ids.VIEW_ALL_NOTIFICATIONS}
-                        id={build(
-                            ids.BASE_DEBUG_ID,
-                            ids.NOTIFICATIONS_MENU,
-                            ids.VIEW_ALL_NOTIFICATIONS
-                        )}
-                        handleClose={handleClose}
-                        isMobile={isMobile}
-                    />,
-                    <IconButton
-                        key={ids.MARK_ALL_READ}
-                        className={classes.markSeen}
-                        onClick={handleMarkAllAsSeenClick}
-                        id={build(
-                            ids.BASE_DEBUG_ID,
-                            ids.NOTIFICATIONS_MENU,
-                            ids.MARK_ALL_READ
-                        )}
-                    >
-                        <DoneAllIcon size="small" />
-                    </IconButton>,
-                ]}
+
+                {!userProfile?.id && (
+                    <ListItem>
+                        <NotAuthorized errorObject={errorObject} />
+                    </ListItem>
+                )}
+
```

Comment:
While this works, `NotAuthorized` is for `HTTP status code 403` error (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403). An example of this error will be a regular user trying to access Admin features in DE. In this case, users are simply trying to access a feature that requires them to login (`Http status code 401`). So creating a component similar to `NotAuthorized` will be useful. Say `NotLoggedIn` ? or We should rename `NotAuthorized` to `Forbidden` and name the new component for `401` to be `Not Authorized`.

Reply:
I've updated it using a similar component `NotLoggedIn` with `status code 401`. Let me know if this is what was expected and I'll merge this.

Reply:
Almost there. Now that #308 is merged, if you look at `src/components/utils/error/errorCode.js`, the `errorObject`  for `401` is defined there as `signInErrorResponse`. So `NotLoggedIn` should be refactored to use that. And then this is good to go!.

Reply:
One more thing before you merge - you have a commit that still has `WIP` in the commit title.  Let's fix that commit message before merging.  

I think there's some optimization that can be done with this component - there are a lot of checks for whether the user is signed in or not, mobile or not, signed in and mobile, signed in and not mobile, etc. but let's address that in the next PR 👍